assets/apps/customizer-controls/src/builder/HFGBuilderComponent.tsx

```diff
@@ -151,10 +155,14 @@ const HFGBuilderComponent: React.FC<Props> = ({ control, portalMount }) => {
 					}
 
 					if (
-						activeSection.id ===
-						'neve_pro_global_page_header_settings'
+						[
+							'neve_pro_global_page_header_settings',
+							'hfg_page_header_layout_top',
+							'hfg_page_header_layout_bottom',
+						].includes(activeSection.id)
 					) {
 						setMounted(true);
+						setHidden(false);
```

Comment:
I don't think listening to sections and unhiding the page header is necessary. I'm not sure why this binding to the expanded section existed in the first place. There can be only one active panel, regardless of what section inside it is active.